src/library/dlgtagfetcher.cpp

```diff
@@ -234,7 +269,21 @@ void DlgTagFetcher::reject() {
 
 void DlgTagFetcher::fetchTagProgress(const QString& text) {
     QString status = tr("Status: %1");
-    loadingStatus->setText(status.arg(text));
+    loadingProgressBar->setFormat(status.arg(text));
+    m_progressBarValue += 15;
```

Comment:
Avoid magic numbers. Can this be expressed by a formular derived from constants?

Reply:
The magic numbers added in order to have a better scaling, before that I was making assumptions about the results coming from AcoustID. This assumptions were causing unwanted behavior such as jumping from 0 to 100 or backward loading from 40 to 30. 

Right now it starts from 10%, then goes to %25 and then %40 between the fingerprinting, identifying and retrieving. 

I will try to do it with the derived from constants.